### PR TITLE
Resolve #1024, #1027, #1031

### LINKED ISSUE #1024: Recipients UI Improvements
1. Add recipient modal text field placeholder color, add button hover bgColor

2. Delete recipient modal warning button hover style

3. Toast style and positioning

### PR DESCRIPTION
**Pull request by williamleha**

### CODE REVIEW

File: src/styles/theme.js [hunk 1]
```diff
@@ -149,13 +166,74 @@ export const themeChainsfr = createMuiTheme({
...
         lineHeight: textValues.default.lineHeight,
         color: textValues.default.color
       }
     },
+    MuiSvgIcon: {
+      fontSizeLarge: {
+          fontSize: 32
+      }
+    },
+    MuiTextField: {
```
Comment (InVincible2016) [inline src/styles/theme.js]: This would case TextField in transferForm to have extra padding

File: src/styles/layout.js [hunk 2]
```diff
@@ -58,6 +58,34 @@ export const componentPaddings = {
...
+    title: {
+      paddingTop: spacing.s,
+      paddingBottom: spacing.m, // MuiDialogTitle and MuiDialogContent share the 20/30px spacing to support textfield label transformation
+      paddingRight: spacing.s   //for close action
+    },
+    contentForm: {
+      paddingTop: spacing.s, // MuiDialogTitle and MuiDialogContent share the 10/30px spacing to support textfield label transformation
+      paddingBottom: spacing.l
+    },
+    contentText:{
```
Comment (InVincible2016) [inline src/styles/layout.js]: space before {

File: src/styles/layout.js [hunk 3]
```diff
@@ -58,6 +58,34 @@ export const componentPaddings = {
...
+      paddingBottom: spacing.l
+    }
+  },
+  textField: {
+    default:{
```
Comment (InVincible2016) [inline src/styles/layout.js]: space before {

File: src/styles/theme.js [hunk 4]
```diff
@@ -149,13 +166,74 @@ export const themeChainsfr = createMuiTheme({
...
     },
+    MuiDialogActions: {
+      root: {
+        paddingTop: componentPaddings.modal.clearfix.none,
+        paddingBottom: componentPaddings.modal.default.paddingBottom,
+        paddingLeft: componentPaddings.modal.default.paddingLeft,
+        paddingRight: componentPaddings.modal.default.paddingRight
+      }
+    },
+    MuiDialogContent:{
```
Comment (InVincible2016) [inline src/styles/theme.js]: space before {

File: src/styles/theme.js [hunk 5]
```diff
@@ -149,13 +166,74 @@ export const themeChainsfr = createMuiTheme({
...
+          paddingBottom: componentPaddings.modal.contentForm.paddingBottom,
+        },
+
+        '&.dialog-content': {
+          paddingTop: componentPaddings.modal.clearfix.none,
+          paddingBottom: componentPaddings.modal.contentText.paddingBottom,
+        }
+      }
+    },
+    MuiDialogTitle:{
```
Comment (InVincible2016) [inline src/styles/theme.js]: space before {

File: src/styles/theme.js (hunk 1 above)
Comment (williamleha) [reply src/styles/theme.js]: fixed

**Pull request merged**